dask_jobqueue/tests/test_jobqueue_core.py

```diff
@@ -427,12 +427,11 @@ def test_security():
         require_encryption=True,
     )
 
-    with LocalCluster(
-        cores=1, memory="1GB", security=security, protocol="tls"
-    ) as cluster:
```

Comment:
I reused the old test to test if a security context is enough to use tls

Reply:
Maybe the tests should be duplicated? Not a big deal though.